NewForce-Cohort-5/tabloidfullstack-fool-s-spring
# Tags - View All Tags

# Description

This is a new feature that allows a user to view all of the available tags in the database through the browser. A user logs in, then navigates to the Tags link in the navigation bar. A list of available tags should then be displayed in alphabetical order. 

### [TICKET #13 View All Tags](https://trello.com/c/aZfIzfXR/13-13-view-all-tags)

_Note: Create/Edit/Delete are non-functioning and will be on additional PRs matching Trello tickets 14-16._

## Type of change


# Testing Instructions

- run `git add .` then `git commit` all changes in your terminal
- `git checkout main` then `git fetch -a` to get all branches that haven't been fetched yet
- `git checkout ajb-tag-view`
- `start ~{working-directory}/tabloidfullstack-fool-s-spring/Tabloid.sln`
- Run these queries in your SQL Object Explorer [Create DB](https://github.com/NewForce-Cohort-5/tabloidfullstack-fool-s-spring/blob/main/SQL/01_Tabloid_Create_DB.sql) and [Seed Data](https://github.com/NewForce-Cohort-5/tabloidfullstack-fool-s-spring/blob/main/SQL/02_Tabloid_Seed_Data.sql)
- In Visual Studio, run in debugger using the `Tabloid` server
- `cd ~{working-directory}/tabloidfullstack-fool-s-spring/Tabloid/client && npm start`
- Log in as 'foo@bar.com'
- Click the Tags link in the navigation bar
- You should now see a list of all tags.

| Id | Name
| --- | --- 
| 1 | C#
| 2 | JavaScript
| 3 | Funny
| 4 | Scary 
| ... | ... 


# Checklist:
